packages/documentation/src/components/Header.jsx

```diff
@@ -93,21 +104,18 @@ export default class Header extends React.Component {
                 </em>
               </div>
               <div className="site-c-header__utility-links">
-                <div id="search-container" className="site-search-container">
-                  <i className="fas fa-search search-icon" />
-                  <input
-                    type="search"
-                    className="site-search-container__input"
-                    id="search-input"
-                    placeholder="Search..."
-                    aria-label="Search"
-                  />
-                  <ul
-                    id="results-container"
-                    className="site-search-results"
-                    role="listbox"
-                  />
-                </div>
+                <StaticQuery
```

Comment:
This component composition is a little messy- I might move the search out of this file and just export a functional component with this composition.